tensorflow/core/util/cuda_kernel_helper.h

```diff
@@ -68,15 +70,14 @@ inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
 template <typename DeviceFunc>
 inline CudaLaunchConfig GetCudaLaunchConfig(int work_element_count,
                                             const GPUDevice& d, DeviceFunc func,
-                                            size_t dynamic_shared_memory_size) {
+                                            size_t dynamic_shared_memory_size = 0) {
```

Comment:
`cudaOccupancyMaxPotentialBlockSize` has a default value for this parameter, so I add the same value here. I will remove it if doing so will make it fits better with the code style of tensorflow.